app/src/main/java/ar/com/wolox/android/example/ui/example/home/IHomeView.java

```diff
@@ -0,0 +1,8 @@
+package ar.com.wolox.android.example.ui.example.home;
+
+/**
+ *
+ */
+public interface IHomeView {
+
```

Comment:
Add a TODO here if it is going to be implemented later

Reply:
✔ **Done**